tests/integration/reconcile_test.go

```diff
@@ -196,8 +199,8 @@ func (s *scaleoutClusterSuite) TestScaleoutClusterSuite() {
 	err = s.tc.TypedObjectClient().Update(
 		ctx, smbShare)
 	require.NoError(err)
+	s.minPods = 3
```

Comment:
> I guess the test already had that flaw though. Maybe what we really want is (pseudocode):
> 
> ```
> updateSmbShare();
> ctx2 := contextWithTimeout()
> poll(ctx, func() {
>   l, err := StatefulSets.List(...)
>   checkStatefulSet(l)
> })
> require.NoError(waitForPodExist(ctx, s), "smb server pod does not exist")
> ```
> 
> What do you think?

Ok, that sounds reasonable.

Reply:
> Would it make more sense to do it like `s.minPods += 1` to make it match the "+1" above rather than assuming 3?

I've used `newClusterSize` for now.